lxd/main.go

```diff
@@ -633,6 +635,10 @@ func setupLXD() error {
 	}
 
 	if *argAuto {
+		if *argStorageBackend == "" {
```

Comment:
Fair enough. I was using it because I wanted to make sure I was setting the
values correctly (when you say "no" to LXD being publicly available I don't
know what actual config values it is setting.)

John
=:->


On Thu, Mar 10, 2016 at 12:51 PM, Stéphane Graber <notifications@github.com>
wrote:

> In lxd/main.go <https://github.com/lxc/lxd/pull/1736#discussion_r55648832>
> :
>
> > @@ -633,6 +635,10 @@ func setupLXD() error {
> >  	}
> >
> >  	if *argAuto {
> > +		if *argStorageBackend == "" {
>
> If you want to change the config after intiial configuration, that's what
> "lxc config set" is for.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lxc/lxd/pull/1736/files#r55648832>.
>